README.md

```diff
@@ -36,52 +41,46 @@ development stage (no stable release available).
 
 To compile LibrePCB, you need the following software components:
 - g++ >= 4.8, MinGW >= 4.8, or Clang >= 3.3 (C++11 support is required)
-- Qt >= 5.2 (http://www.qt.io/download-open-source/)
-- `libglu1-mesa-dev` (`sudo apt-get install libglu1-mesa-dev`)
-- `zlib` / `zlib-dev` (http://www.zlib.net/)
-- `OpenSSL` (https://www.openssl.org/)
+- [Qt](http://www.qt.io/download-open-source/) >= 5.2
+- [zlib](http://www.zlib.net/)
+- [OpenSSL](https://www.openssl.org/)
 
-#### Installation on Ubuntu 14.04 and later
+#### Installation on Debian/Ubuntu/Mint
 
 ```bash
-sudo apt-get install g++ qt5-default qttools5-dev-tools qt5-doc qtcreator libglu1-mesa-dev zlib1g zlib1g-dev openssl
+sudo apt-get install g++ qt5-default qttools5-dev-tools qt5-doc qtcreator \
+libglu1-mesa-dev zlib1g zlib1g-dev openssl
 ```
 
-#### Installation on ArchLinux
-
-You can install [librepcb-git](https://aur.archlinux.org/packages/librepcb-git/) from the AUR.
```

Comment:
OK, I replaced `gcc` and `make` by `base-devel`.

> To quote the packaging guidelines:
> > Note: The group base-devel is assumed to be already installed when building with makepkg.

That may be true, but it's not relevant for the dependency list in our readme, because when building LibrePCB "by hand" (not by the `librepcb-git` package from AUR) one needs to install `base-devel` by himself.

So actually the dependency list of the AUR package is different by the one we provide in our readme because these are two different use cases.